diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 4b64c8a6e8d5a66321889467a06abb15033a57ac..1f96b48768d6d37373a170a2e4afb100e39baa4d 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -346,6 +346,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config/compiler/pgo:default_pgo_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
