diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index e9b664b7d28681f81dceb2d7c2ce5d48fbb31b6d..66f82d1e29cb10fe087b6f1ac0a6ae9287b4714f 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -91,6 +91,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window/internal" ]
 }
 
 source_set("unit_tests") {
