diff --git a/chrome/browser/resources/settings/basic_page/basic_page.ts b/chrome/browser/resources/settings/basic_page/basic_page.ts
index d02b76c503d95237838c2447b6499b6e85054364..c045e702ae69e629f4447e61a74d47441b242bae 100644
--- a/chrome/browser/resources/settings/basic_page/basic_page.ts
+++ b/chrome/browser/resources/settings/basic_page/basic_page.ts
@@ -54,7 +54,8 @@ export class SettingsBasicPageElement extends SettingsBasicPageElementBase {
 
   /** Overrides MainPageMixin method. */
   override containsRoute(route: Route|null): boolean {
-    return !route || routes.PRIVACY.contains(route);
+    return !route || routes.BASIC.contains(route) ||
+        (routes.ADVANCED && routes.ADVANCED.contains(route));
   }
 
   // SettingsViewMixin implementation.
