diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index b4e4aa7dc1c915c6af08675534d265f1747d4f7c..7d3a0a18ffdb34fe8f9bc89d28e944485598f4d8 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -88,6 +88,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window/internal" ]
 }
 
 source_set("unit_tests") {
