diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index cf976521ee6905d1740a32dbe9dee9bc614c4d2a..531e6920b1bcc4234ca0871303ccc4a052bf0418 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4859,6 +4859,7 @@ constexpr char kWebiumFlag[] = "webium";
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
