diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 099f036a046dc1e1dfadc93393842c973b73259c..89828dfab6e5f59c18e943da55567ea651e62616 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1360,6 +1360,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // via `HasContentSettingChangedViaPageInfo(type)`.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
