diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 48392f347d5e8c974b4a8cfda448ed1f49c49a2e..a97c0a3cfa96e8c92c70c030796102ba651617be 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3681,6 +3681,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
