diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 017d9d34f544fb5a3ab6d44fa90d21b878ac8ef2..dce0e63be964e433c67c8ffcad0e73fa77901d00 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -748,6 +748,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
