diff --git a/third_party/android_deps/autorolled/BUILD.gn b/third_party/android_deps/autorolled/BUILD.gn
index 2d06e0ae64ff15abbd9f1ee4a2d4b017e979feb3..c2c9ca73e785f6ccee21cf3db6993d7989cd3d95 100644
--- a/third_party/android_deps/autorolled/BUILD.gn
+++ b/third_party/android_deps/autorolled/BUILD.gn
@@ -840,6 +840,7 @@ template("define_autorolled_targets") {
           "//third_party/android_deps:google_android_datatransport_transport_runtime_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
@@ -864,6 +865,7 @@ template("define_autorolled_targets") {
           "//third_party/android_deps:javax_inject_javax_inject_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
