diff --git a/ui/native_theme/native_theme_win.cc b/ui/native_theme/native_theme_win.cc
index 07d34df0490db28caeb14982072c84cb6e369c59..47858797248dea5c9f18124f1aefde35d7e3013c 100644
--- a/ui/native_theme/native_theme_win.cc
+++ b/ui/native_theme/native_theme_win.cc
@@ -1696,6 +1696,7 @@ void NativeThemeWin::UpdateDarkModeStatus() {
                                     &system_uses_light_theme);
     system_dark_mode_enabled = (system_uses_light_theme == 0);
   }
+  BRAVE_NATIVETHEMEWIN_UPDATEDARKMODESTATUS
   set_use_dark_colors(dark_mode_enabled);
   set_use_dark_colors_for_system_integrated_ui(system_dark_mode_enabled);
   set_preferred_color_scheme(CalculatePreferredColorScheme());
