diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
index 701b806c2e8122d4767afb1bd21bd211c7ca23d8..6c73be94dac8e3b413cb2de137c03d5d1f0b9dc0 100644
--- a/front_end/panels/network/NetworkItemView.ts
+++ b/front_end/panels/network/NetworkItemView.ts
@@ -386,3 +386,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
     return this.#headersViewComponent;
   }
 }
+import { PatchNetworkItemView } from './NetworkItemView.patch.js'; (NetworkItemView as any) = PatchNetworkItemView(NetworkItemView);
